engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -89,10 +89,19 @@ <h4 translate>Subscriptions</h4>
 
       <dl class="dl-horizontal dl-horizontal-left">
         <dt translate>Subscription Status</dt>
-        <dd>
+        <dd ng-if="!simpleContentAccessEnabled">
           <i ng-class="getHostStatusIcon(host.subscription_global_status)"></i>
           {{ host.subscription_status_label }}
         </dd>
+        <dd ng-if="simpleContentAccessEnabled">
+          <!-- always display green icon for SCA -->
+          <i
+            ng-class="getHostStatusIcon(0)"
```

Comment:
Instead of doing this you can update getHostStatusIcon to understand `5`  

https://github.com/Katello/katello/blob/bb672414d89e51eee88453ea73b91d1250594343/app/models/katello/subscription_status.rb#L3

Reply:
`getHostStatusIcon` is called with `host.subscription_global_status` which calls `to_global`, which only gives values from `::HostStatus::Global::ERROR`, `OK`, or `WARN` (0-2).  The '5' / `DISABLED` is returned not from `to_global`, but from `to_status`, which is then translated in `to_global` to `::HostStatus::Global::OK`.  All that being said, I don't think it makes sense to add a `5` value to `getHostStatusIcon`, since those values line up with the `to_global` return values.

Reply:
Updating to remove the `ng-if`, since the only thing we'll lose is the tooltip.  We get the correct green icon already thanks to the `DISABLED` status.